docker/docker-compose-mongodb.yaml

```diff
@@ -15,17 +15,20 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# note that the project needs to be built once with `mvn clean install`
+# note that the project needs to be built once with `mvn clean install -Ddocker.skip=false`
 services:
     sling:
         image: apache/sling:snapshot
         ports:
-            - "8080"
+            - "8080:8080"
+            # when SLING_DEBUG=true
+            # - "5005:5005"
```

Comment:
I think we can always leave the port mapping on. If the debugger is not enabled there is no harm done.

Reply:
done